pkg/admission/admission.go

```diff
@@ -256,6 +258,9 @@ func (a *Admission) serve(w http.ResponseWriter, r *http.Request, admit admitFun
 
 	responseAdmissionReview.Response.UID = requestedAdmissionReview.Request.UID
 
+	responseAdmissionReview.APIVersion = requestedAdmissionReview.APIVersion
+	responseAdmissionReview.Kind = requestedAdmissionReview.Kind
+
```

Comment:
I'm not sure if the second function "serve" is actually used. It may be dead code.

Reply:
indeed 

```
$ golangci-lint run --disable-all -E unused ./pkg/admission/.
pkg/admission/admission.go:224:21: func `(*Admission).serve` is unused (unused)
```

I've filed https://github.com/prometheus-operator/prometheus-operator/issues/3576 as a follow-up.